### DISCUSSION TITLE
New keyboard shortcuts assume an ANSI compatible terminal.

### DISCUSSION THREAD

**Discussion by p-gen** [ksh93/ksh#452] [General]
From what I understand from the editing code, it seems that some vt100 escape sequences are hard-coded.
Would it be possible to check their availability before assuming they are usable (by asking for Device Attributes for example)?

And a big thank you for the work already done on ksh93.

**Comment by McDutchie** [comment]
I would happily accept a pull request if someone could make that happen, but I'm pretty sure it's not top on the list for anyone currently working on this code base; there are many higher priority issues still to solve and none of us are getting paid.

What is the harm if those sequences are not usable, anyway? They're input sequences generated by terminal programs as the user presses special PC keyboard keys, so the shell never actively generates them. I'm pretty sure those sequences don't conflict with anything else out there.
